src/Hosting/Hosting/test/WebHostBuilderTests.cs

```diff
@@ -96,7 +96,7 @@ public async Task MultipleUseStartupCallsLastWins(IWebHostBuilder builder)
             var server = new TestServer();
             var host = builder.UseServer(server)
                               .UseStartup<StartupCtorThrows>()
-                              .UseStartup(context => throw new InvalidOperationException("This doesn't run"))
+                              .UseStartup<object>(context => throw new InvalidOperationException("This doesn't run"))
```

Comment:
I'm guessing we're ok with this slight usability regression. The generic arg will always need to be explicitly specified because it can't be inferred from the return type.

Reply:
That's right. I don't see this being common.